bluetoe/link_layer/include/bluetoe/peripheral_latency.hpp

```diff
@@ -0,0 +1,318 @@
+#ifndef BLUETOE_LINK_LAYER_PERIPHERAL_LATENCY_HPP
+#define BLUETOE_LINK_LAYER_PERIPHERAL_LATENCY_HPP
+
+#include <bluetoe/ll_meta_types.hpp>
+
+/**
+ * @file bluetoe/peripheral_latency.hpp
+ *
+ * This file provides several options, how Bluetoe will implement peripheral latency. Peripheral latency
+ * allows a link layer peripheral to miss some connection events (do not respond to the central) to
+ * conserve power.
+ *
+ * Each option is a compromise between latency and power consumption.
+ *
+ * @sa bluetoe::link_layer::peripheral_latency_ignored
+ * @sa ...
+ */
+namespace bluetoe {
+namespace link_layer {
+
+    namespace details {
+        struct peripheral_latency_meta_type {};
+    }
+
+    /**
+     * @brief detailed options for the peripheral latency behavior
+     *
+     * Every option defines a set of events / circumstances under which
+     * the link layer should listen to the very next connection event.
+     * (beside the minumum events, the link layer has to listen to due
+     * to the periperhal latency value of the connection).
+     *
+     * @note in combination, there are some options that make no sense.
+     *       For example, using listen_always with some of the other
+     *       listen_* options is pointless. Also combining send_always
+     *       with send_pending_data doesn not make sense.
+     */
+    enum class peripheral_latency
+    {
+        /**
+         * @brief listen at all connection events.
+         *
+         * This does not imply, that the link layer will also respond at every
+         * connection event.
+         */
+        listen_always,
+
+        /**
```

Comment:
- unacknowledged (several places)
- If a PDU was transmitted, but no acknowledgement was yet received
Enabling this option helps clearing the internal bluetoe tx queue as soon as possible, because bluetoe must keep the last transmitted message in the queue as long as it hasn't been acknowledged.

Reply:
Ok, I hope, I found all occurrences of the misspelling.